packages/shorebird_cli/lib/src/auth/auth.dart

```diff
@@ -126,6 +128,9 @@ class Auth {
       final codePushClient = _buildCodePushClient(httpClient: this.client);
 
       final user = await codePushClient.getCurrentUser();
```

Comment:
Thoughts on caching the user instead of the email so that we don't need to call `getCurrentUser()` in other places like cancel_subscription?